release_contributions_catalog.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+source release_base_script.sh
+
+###### CONTRIBUTIONS CATALOG ######
+docker build --pull -f contributions/Dockerfile -t ${PROJECT_NAME}/contributions-catalog:${VERSION} .
```

Comment:
Shouldn't this build catalog/Dockerfile? Or am I missing something?

Reply:
It was /catalog/Dockerfile. After merging, in the develop branch, /catalog is there. I'm not sure why it's not showing up here.

Reply:
Yes, I think I realized that and created another PR to fix it. The one that was merged today.